shared/ClrFacade/ClrFacade.csproj

```diff
@@ -26,6 +26,13 @@
   <PackageReference Include="R.NET" Version="1.8.1" />
   </ItemGroup>
   <Target Name="PostBuild" AfterTargets="PostBuildEvent">
-    <Exec Command="xcopy /Y $(TargetDir)*.dll $(SolutionDir)..\inst\lib\&#xD;&#xA;xcopy /Y $(SolutionDir)RSharp.runtimeconfig.json $(SolutionDir)..\inst\lib\" />
+    <ItemGroup>
+      <AssemblyFiles Include="$(TargetDir)*.dll" />
+      <ConfigFiles Include="$(SolutionDir)*.runtimeconfig.json" />
+    </ItemGroup>
+    <Copy 
```

Comment:
Nice! this appears to not be very widely used. Most people are doing xcopy and such and that mostly works for windows only solutions.